src/pages/index.js

```diff
@@ -20,88 +64,191 @@ const cardSection = new Section(
   ".gallery__cards"
 );
 
-// Render initial cards
-cardSection.renderItems();
+// Fetch API cards and render them inside the same section
+api.getInitialCards()
+.then((cards) => {
+  cards.forEach((cardData) => {
+    const cardElement = createCard(cardData); // Generate card using createCard
+    cardSection.addItem(cardElement); // Properly add to `.gallery__cards`
+    const likeBtn = document.querySelector('.card__like-button');
+    console.log(cardData.isLiked)
+    if(cardData.isLiked){
+      console.log(cardData._id);  
+      likeBtn.classList.toggle('card__like-button--active');
+    }else{
+      likeBtn.classList.remove('card__like-button--active');
```

Comment:
Please, delete the likes handling from the `api.getInitialCards()` request

Now only the `Card` class knows every **card selector**, so you need to encapsulate the likes logics inside its methods and call them in `index.js`. It’s a violation of **Single responsibility** principle


Everything should be checked only inside the `Card` class